### DISCUSSION TITLE
Ventura-Monterey-BigSur-OC-0.8.7-Release-Infinix-InBook-Series

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#16] [Discuss about the release]
- Updated EFI from Opencore - 0.8.5 --> 0.8.7 .
- Reworked EFI.
- **Note** : I didn't patch USBs and Key mapping. Due to we released the EFI long ago. So, this will fix some issues. Don't worry about patching. I am working on it. I may release it on the next release.
- Just replace your SMBIOS to your personal SMBIOS. Thats it.
- [A Poll, Do you want all Infinix inbook series support (telegram Group) ? 
Anser via here..](https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/discussions/15)

<hr /><em>This discussion was created from the release <a href='https://github.com/devboloji/Infinix-Hackintosh-Opencore-Guide/releases/tag/v0.8.7'>Ventura-Monterey-BigSur-OC-0.8.7-Release-Infinix-InBook-Series</a>.</em>

**Comment by extasea** [comment]
The bluetooth is not working in this release, anything else running smoothly and more stabil than previous release

**Reply by devboloji** [reply]
@extasea Bluetooth is working for mine. Do a resrt nvram once. And please check the Bluetooth kexts in your EFI. Or update the Bluetooth kexts if you are in ventura.

**Comment by ghost** [comment]
Infinix INBook X1 Pro

**Big Sur: I haven't tested it yet**
**Monterey:**  Working fine, bootable and functional, you just need to fix the speaker by changing the alc to **alcid=69**
**Ventura:** Same as **Monterey**, you just need to fix the speaker by changing the alc to **alcid=69**

**CPU**
Intel Core i7-1065G7

**GPU**
Intel Iris Plus Graphics G7

**RAM**
16 GB LPDDR4x On Board
3200 MHz
